packages/kbn-cli-dev-mode/src/base_path_proxy_server.ts

```diff
@@ -21,8 +20,7 @@ import { DevConfig, HttpConfig } from './config';
 import { Log } from './log';
 
 const ONE_GIGABYTE = 1024 * 1024 * 1024;
-const alphabet = 'abcdefghijklmnopqrztuvwxyz'.split('');
-const getRandomBasePath = () => sampleSize(alphabet, 3).join('');
+const getRandomBasePath = () => `spencer`;
```

Comment:
I think we should just make this real.

```suggestion
const getRandomBasePath = () => Math.random() * 100 < 1 ? 'spalger' : sampleSize(alphabet, 3).join('');
```

Also, update test then: (https://github.com/elastic/kibana/blob/a07fb3063b13ed0990a06fd8d3a68ce8b466dbae/packages/kbn-cli-dev-mode/src/integration_tests/base_path_proxy_server.test.ts#L84)

```
  test('root URL will return a redirect location with exactly 3 characters that are a-z (or spalger)', async () => {
    const res = await proxySupertest.get('/');
    const location = res.header.location;
    expect(location).toMatch(/spalger|[a-z]{3}/);
  });
```